op-chain-ops/genesis/config.go

```diff
@@ -476,6 +480,17 @@ func (d *UpgradeScheduleDeployConfig) InteropTime(genesisTime uint64) *uint64 {
 	return offsetToUpgradeTime(d.L2GenesisInteropTimeOffset, genesisTime)
 }
 
+func (d *UpgradeScheduleDeployConfig) L2BlobTime(genesisTime uint64) *uint64 {
+	if d.L2GenesisBlobTimeOffset == nil {
```

Comment:
Why don’t we reuse the offsetToUpgradeTime function here?

Reply:
Good point, done in f42923feae74d608da56912bdb12da69e0862891, also done for op-es branch [here](https://github.com/QuarkChain/optimism/pull/39).